src/main/java/seedu/trippie/Parser.java

```diff
@@ -21,7 +21,7 @@ public static Command parse(String userInput) {
         try {
             if (userInput.equals("exit")) {
                 return new ExitCommand();
-            } else if (userInput.startsWith("buy ")) {
+            } else if (userInput.startsWith("buy")) {
```

Comment:
I removed it so that if it detects eg "buys", it will print an error message and an example on how they can use the [buy] command, what do you all think?